llmlite/llms/chatglm.py

```diff
@@ -99,27 +104,29 @@ def chat(
 
         # TODO: support vllm
         if self.version == chatglm2 and self.backend == consts.BACKEND_VLLM:
-            pass
+            if len(messages) == 1:
```

Comment:
maybe we no longer need to support vllm here, because it will be completed by VLLMBackend. Also remove the TODO.